server-connect/pom.xml

```diff
@@ -32,6 +32,14 @@
 
     <properties>
         <docker.skip-build>false</docker.skip-build>
-        <docker.skip-test>false</docker.skip-test>
+        <docker.skip-test>true</docker.skip-test>
```

Comment:
As mentioned above(https://github.com/confluentinc/kafka-images/pull/211#issuecomment-1557606216), we have added JAVA ITs for the new image. To run those ITs we had to change the maven phase to verify in the jenkinsfile.

Changing the phase to verify will try to run tox ITs. For some of the images, the skip test was marked as false, even though there were no tests added for them.
It was not creating any issue till now because the maven phase was package.

Hence, to run our ITs we had to make this change. It is not affecting anything.